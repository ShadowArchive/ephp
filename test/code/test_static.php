<?php

static $c = 100;

function greetings() {
    static $a = 10, $b = 20;
    print "($a) ($b) hello world\n";
    $a += 10;
    $b *= 2;
}

greetings();
greetings();
greetings();
