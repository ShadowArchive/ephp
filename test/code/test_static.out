(10) (20) hello world
(20) (40) hello world
(30) (80) hello world
